kotlin-android/src/test/java/com/segment/analytics/kotlin/android/EventsFileTest.kt

```diff
@@ -22,7 +22,8 @@ import java.util.*
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 class EventsFileTest {
     private val epochTimestamp = Date(0).toInstant().toString()
-    private val sharedPreferences = MemorySharedPreferences()
+    private val _sharedPreferences = MemorySharedPreferences()
+    private val kvStore = AndroidKVS(_sharedPreferences)
```

Comment:
I noticed you used _sharedPreferences for shadow var. We often do this to signify to "not touch" on the obj-c side. Do we want to apply underscore to kvStore as well?

Reply:
i think this was a refactoring issue. normally ur suggestion would apply, but this is a test file so im not sure if the concept of shadow var / "do not touch" applies here